fenix/app/nimbus.fml.yaml

```diff
@@ -318,6 +318,14 @@ features:
         type: Map<String, Boolean>
         default: {}
 
+  splash-screen:
+    description: "A feature that sets the maximum duration to display the splash screen during the first launch."
```

Comment:
Bikeshedding: I would talk about the feature (i.e. the splash screen, and it's only briefly visible at first launch) rather than the maximum duration of the display.

Also: consider having an `enabled` or `is-enabled` boolean here too.

Reply:
@jhugman 
How about the updated version?